### PR TITLE
[IMP] product_secondary_unit: Allow skip set default value for secondary_uom_qty

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT43104

ping @ernestotejeda @carlosdauden

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: product_secondary_unit/models/product_secondary_unit_mixin.py [hunk 1]
```diff
@@ -131,3 +130,12 @@ def _onchange_helper_product_uom_for_secondary(self):
             precision_rounding=self.secondary_uom_id.uom_id.rounding,
         )
         self.secondary_uom_qty = qty
+
+    @api.model
+    def default_get(self, fields_list):
+        defaults = super().default_get(fields_list)
+        if self.secondary_uom_id and not self.env.context.get(
```
Comment (ernestotejeda) [inline product_secondary_unit/models/product_secondary_unit_mixin.py:137]: Is it necessary to check `self.secondary_uom_id`?

Comment (sergio-teruel) [reply product_secondary_unit/models/product_secondary_unit_mixin.py:137]: I ithink yes... I do not set a default value to 1.0 if not secondary uom set...

**Review by BT-aleonard — APPROVED** [review]
LGTM,
Thanks.

**Pull request merged**